plugins/spamassassin.js

```diff
@@ -20,11 +24,11 @@ exports.load_spamassassin_ini = function () {
         max_size:     500000,
         old_headers_action: "rename",
         subject_prefix: "*** SPAM ***",
-        bypass_extra_headers: false,
+        add_headers: true,
     };
 
     for (var key in defaults) {
-        if (plugin.cfg.main[key]) continue;
+        if (plugin.cfg.main[key] !== undefined) continue;
         plugin.cfg.main[key] = defaults[key];
```

Comment:
this is an error and will prevent most of the defaults from being initialized. Please revert.